modules/messages/src/mock.rs

```diff
@@ -402,8 +402,10 @@ pub fn message(nonce: MessageNonce, payload: TestPayload) -> Message {
 }
 
 /// Return valid outbound message data, constructed from given payload.
-pub fn outbound_message_data(payload: TestPayload) -> MessagePayload {
-	payload.encode()
+pub fn outbound_message_data<T: Config<I>, I: 'static>(
```

Comment:
Nit: you could remove generic args and just return `StoredMessagePayload<TestRuntime, ()>`

Reply:
Done